x-pack/plugins/security_solution/common/api/endpoint/actions/get_file.schema.yaml

```diff
@@ -30,7 +30,6 @@ components:
         - type: object
           required:
             - parameters
-            - file
```

Comment:
Was this removed on purpose?

Reply:
yes. The `file` param is redundant. The schema needs a `parameters` only.